Display the eval buffer using the default display-buffer-alist (#29)
tminor/jsonnet-mode: README.md

- `jsonnet-use-smie` enables SMIE-provided indentation.
- `jsonnet-indent-level` changes the number of spaces used to indent
  Jsonnet code.

## Buffer display

The potential buffers that the mode creates are:

* `*jsonnet output*`
* `*jsonnetfmt stderr*`

> * `*jsonnetfmt stdout*`

Author: ah, i didn't see that. but you're right, i'll remove them

> 
> If you require special customisation of their display, you ca either
> use the `display-buffer-alist` as described in a previous section or
> investigate the options provided in your distribution of GNU Emacs.

Reviewer: Another pedantic suggestion:

```suggestion

If you require special customization of their display, you can either
use `display-buffer-alist` as described [above](#buffer-display) or
investigate the options provided in your distribution of GNU Emacs.
```
